views/gym-routes.html

```diff
@@ -501,6 +517,19 @@ <h3>{{ reference_data['current_language'].gyms }} </h3>
       
           // Create the Tabulator table
           var table = new Tabulator("#tabulator-table", {
+            printHeader:"<h1>SKALA3MA - {{ gym['name'] }}</h1>", // set header content on printed table
```

Comment:
can you add the route name as well (so that we know the version of the printer routes)

Reply:
done